adapters/postgres/postgres.go

```diff
@@ -518,7 +518,15 @@ func (adapter *Postgres) SelectFields(fields []string) (sql string, err error) {
 		return
 	}
 	var aux []string
+
 	for _, field := range fields {
+		groupFunc, _ := NormalizeGroupFunction(field)
```

Comment:
This is something that eventually we should improve because this `NormalizeGroupFunction` strict us to use some small fraction of the all Postgres aggregate functions and to me it is pretty bad.

Even worse it allow us to use `median` function that is not defined in Postgres builtin functions:
```bash
prest on  main on 🐳 v20.10.21 
➜ curl 'http://localhost:3000/prest/public/foo?_select=c1,median:c2&_groupby=c1'
{
	"error": "pq: function median(double precision) does not exist"
}
```